src/main/java/com/Kh033Java/travelbook/repository/DescriptionRepository.java

```diff
@@ -1,9 +1,19 @@
-package com.Kh033Java.travelbook.repository;
-
-import org.springframework.data.neo4j.repository.Neo4jRepository;
-
-import java.util.List;
-
-public interface DescriptionRepository extends Neo4jRepository<DescriptionRepository, Long> {
-    List<DescriptionRepository> findAll();
-}
+package com.Kh033Java.travelbook.repository;
+
+import org.springframework.data.neo4j.annotation.Query;
+import org.springframework.data.neo4j.repository.Neo4jRepository;
+import org.springframework.data.repository.query.Param;
+
+import com.Kh033Java.travelbook.entity.Country;
+import com.Kh033Java.travelbook.entity.Description;
+
+import java.util.List;
+
+public interface DescriptionRepository extends Neo4jRepository<DescriptionRepository, Long> {

```

Comment:
change from `<DescriptionRepository, Long>` to `<Description, Long>`

Reply:
done